src/panels.h

```diff
@@ -16,6 +16,11 @@ class JsonOut;
 class avatar;
 struct point;
 
+namespace activity_level
+{
+const std::string activity_level_str( float level );
+}
```

Comment:
```suggestion
} // namespace activity_level
```
Namespace require terminating with a closing comment.